fundamentals/higher-order-functions-drills.md

```diff
@@ -94,4 +94,4 @@ rocksWarning('Centinela Ave and Olympic Blvd');
   * Otherwise, capitalize the LAST character of the array element and add it to your accumulator
 * Your input is `'noggin oreo the moon time tele steed his tent apollo her lives though shoo tofu budapest'`
   * You will need to convert the input to an array before using `.reduce()`
-* **HINT**: When you invoke `reduce()` you will need to set the `initialValue` parameter to an empty string so that future iterations can concatenate more string characters 
\ No newline at end of file
+* **HINT**: When you invoke `reduce()` you will need to set the `initialValue` parameter to an empty string so that future iterations can concatenate more string characters 
```

Comment:
What's the diff on this line?  I can't see it!

Reply:
Hmm, I used the web interface. Looks like there was previously no newline at the end of the file, and it put one in.